examples/colors/store.rs

```diff
@@ -0,0 +1,193 @@
+use glib::{ParamFlags, ParamSpec, Value};
+use gtk::gdk;
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+use crate::color::Color;
+
+const n_colors: usize = 256 * 256 * 256;
+
+mod imp {
+    use std::cell::RefCell;
+
+    use super::*;
+    use gtk::glib::once_cell::sync::Lazy;
```

Comment:
once_cell is already defined as a dependency in the examples's Cargo.toml; use that one?